### DISCUSSION TITLE
Sidebar buttons

### DISCUSSION THREAD

**Discussion by Glitchy-Tozier** [8VIM/8VIM#121] [Ideas]
1. We already have great shortcuts for selecting text, copying, pasting, for cutting, etc… I'm not sure whether we actually need that `CTRL`-button. Could someone explain why it still exists?
I feel like it just wastes space (and kind of violates that one-handedness-paradigma).
2. Sometimes `CTRL C` randomly doesn't work. No idea why.

**Comment by flide** [comment]
I think a better discussion would be : Which 5 buttons should be visible on the sidebar as of now.

Emoji, and selection keyboard switcher are there to stay. Setting is also good to have.
what would you guys like instead of tab and ctrl?

**Comment by Glitchy-Tozier** [comment]
>Emoji, and selection keyboard switcher are there to stay. Setting is also good to have.

I agree, although the necessity of the settings-button is debatable.
Two thoughts:
1. Having only FOUR buttons sounds better, now that #115 is done. (because the keyboard is less high now)
2. To answer your question: As you know, I currently really like the idea of being able to input special characters easier. So this is a nice idea I think:

![8vim-drawing](https://user-images.githubusercontent.com/59611881/101058711-b64b9680-358d-11eb-8338-b56d6110e879.jpg)
